activities/chris.py

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env python3
+
+import os
+
+import grass.script as gs
+
+
+def run_curvatures(scanned_elev, env, **kwargs):
+    gs.run_command(
+        "r.param.scale",
+        input=scanned_elev,
+        output="profile_curv",
+        method="profc",
+        size=11,
+        env=env,
+    )
+    gs.run_command(
+        "r.param.scale",
+        input=scanned_elev,
+        output="tangential_curv",
+        method="crosc",
+        size=11,
+        env=env,
+    )
+    gs.run_command(
+        "r.colors", map=["profile_curv", "tangential_curv"], color="curvature", env=env
+    )
```

Comment:
Only one of these can be displayed at a time, so either pick one, or a more unique solution is to combine them together for visualization. One option is to use one or the other based on row and column number (command line syntax):

```python
gs.run_command("r.mapcalc", expression="curvature_mix = if(row() > col(), tangential_curv, profile_curv)")
```

(The split would be a diagonal if the computational region is a square.)